lib/cmake/libcudacxx/libcudacxx-config-version.cmake

```diff
@@ -26,7 +26,9 @@ set(PACKAGE_VERSION_EXACT FALSE)
 set(PACKAGE_VERSION_UNSUITABLE FALSE)
 
 if(PACKAGE_VERSION VERSION_GREATER_EQUAL PACKAGE_FIND_VERSION)
-  set(PACKAGE_VERSION_COMPATIBLE TRUE)
+  if(libcudacxx_VERSION_MAJOR VERSION_EQUAL PACKAGE_FIND_VERSION_MAJOR)
```

Comment:
I believe that libcudacxx wants compatibiltiy across major version values.

Reply:
@wmaxey @jrhemstad Come up with a truth table for how requested version and found version should interact for major version 1, 2, and 3+ and I will encode it.

Reply:
Thanks, that's what I thought, but good to have it explicit. Requested in rows, found in columns, right?

Reply:
> Requested in rows, found in columns, right?

Yep!